dictionaries/en_shared/src/shared-additional-words.txt

```diff
@@ -1,55 +1,103 @@
 # These are shared words for all variants of English.
+ADHD
 aesthetical
 Alpha Centauri
 amphitheatre
-anthropomorphization
+archivability
 archivable
 archivables
 Aruban
 Arubans
+attaboys
+audiobook
+audiobooks
 bicep
 Bonanno
+breathwork
+caprese
 cellpadding
 CEST
+cheong
 Christoph
 cohabitate
 cohabitated
 cohabitates
 cohabitating
 cohabitator
 cohabitators
+comfortability
 conformant
 counterparties
 counterparty
 COVID
 crafter
+crema
+decolonial
+Diablo
 diegetically
 disambiguates
 dropshipper
 dropshippers
 dropshipping
 drumless
 dryas
+ecommerce
 EET
 eligibilities
 endoliths
+esports
 futhorc
+gameplay
+gaslighted
+gaslighter
+gaslighters
+gaslighting
+ghanouj
+girlboss
 guillemet
+guilted
+guilting
 haikus
 haptic
 haptics
 heatmap
 hmmm
 hydrophile
 hydrophobe
+hyperaware
+hyperfocus
+hyperfocused
+hyperfocuses
+hyperfocusing
 hyponym
 hyponyms
+indigeneity
 iPads
 leaderboard
 leaderboards
+liberatory
+longform
+losslessly
+Luddism
+ludology
 marionnettes
+micellar
 minimalistic
 mitigations
+mukimame
+notetaker
```

Comment:
Cool, I assumed it was just "note taker", but it looks like the other form is quite common: [notetaker vs note taker](https://books.google.com/ngrams/graph?content=notetaker%2Cnote+taker&year_start=1990&year_end=2022&corpus=en&smoothing=3&case_insensitive=false)